src/content/docs/usage/prebuilds.mdx

```diff
@@ -0,0 +1,189 @@
+
+Prebuilds achieve this by setting up a [project configuration](/usage/projects#project-configuration) that monitors changes in the connected Git Provider's repository.
+
+Prebuilds work by registering a listener for webhook events from the Git Provider. A public API endpoint is provided, which the Git Provider uses to send these requests.
+
+## Add a Prebuild
+
+1. Run the following command to add a new Prebuild:
+
+   ```shell
+   daytona prebuilds add
+   ```
+
+   <Aside type="note">
+      Add a `--run` flag to the command to automatically run the build after creating the Prebuild.
+   </Aside>
+
+   <br />
+
+   Upon running the command, Daytona will prompt you to select a project configuration you plan to work on. You can then decide on a commit interval (e.g. `5`, _every 5 commits_) after which a build should be triggered, any specific trigger files whose changes should immediately start the build process, and build retention (the maximum number of builds to store at a time).
+
+   ```text
+   Select a Project Config To Prebuild
+   1 item
+   MyConfig
+   https://github.com/username/myproject.git
+   ```
+
+   ```text
+   Commit interval
+   Leave blank to ignore push events
+   >
+   ```
+
+   ```text
+   Trigger Files
+   Enter full paths for files whose changes you want to explicitly trigger a prebuild. Use new lines for multiple entries.
+   ```
+
+   ```text
+   Retention
+   Maximum number of resulting builds stored at a time
+   >
+   ```
+
+Subsequent `daytona create` calls will automatically detect the most recent existing build and use it to create the project.
+
+## Use a Prebuild
+
+Once you added a Prebuild, it becomes active immediately based on the [project configuration](usage/projects#project-configuration) you provided. The Prebuild will automatically run whenever the specified conditions are met, such as when the defined number of commits is reached. For instance, if you specified a commit interval of `5`, the Prebuild will run after every `5` commits to the repository.
+
+As soon as you add a Prebuild, a build process will automatically start in the background unless you have configured it otherwise. The initial build process may take some time to complete, but subsequent builds will be significantly faster. However, if you try to create a Workspace before this build is completed, Daytona will start a fresh build, which might take longer.
+
+It is recommended to wait (e.g., 10 minutes) for the initial build to complete before creating a Workspace.
+
+You can now create a Workspace using the Prebuild by running the following command:
+
+```shell
+daytona create
+```
+
+Upon running this command, Daytona will detect the most recent Prebuild associated with the selected project configuration. The Workspace will then use this Prebuild, significantly speeding up the setup process.
```

Comment:
I'd add "by utilizing the cached environment." to the end of this sentence and remove the one after it ("During... slowing down the workspace creation")